documentation/asciidoc/topics/yaml/backup.yaml

```diff
@@ -4,3 +4,6 @@ metadata:
   name: my-backup
 spec:
   cluster: source-cluster
+  volume:
+    storage: 1Gi
+    storageClassName: "blah"
```

Comment:
You forgot :stuck_out_tongue_winking_eye:

Reply:
nuts. the one thing...